mappings/in-20100104-2258#in-20091231-2255.tinydiff

```diff
@@ -508,5 +508,5 @@ c	net/minecraft/unmapped/C_8425960
 		p	1		p_1	title
 	m	(Ljava/lang/String;)V	m_3915830	m_3915830	progressStage
 		p	1		p_1	stage
-	m	(I)V	m_1991224	m_1991224	progressPercentage
+	m	(I)V	m_1991224	m_1991224	progressStagePercentage
```

Comment:
I'm not against this change per se, but the ProgressListener/ProgressRenderer exists in a wide range of versions so if/when the change is made it should be applied to all at once.